.github/workflows/release.yaml

```diff
@@ -1,18 +1,18 @@
 name: release
 
 on:
-  push:
-    branches:
-      - main
-    paths:
-      - CHANGELOG.md
-  workflow_dispatch:
+  workflow_call:
```

Comment:
i don't think we should change this to workflow call since this isn't supposed to be a reusable workflow